File: fm-domain/src/main/java/io/zeitwert/fm/util/CustomFilters.java

package io.zeitwert.fm.util;

import io.crnk.core.queryspec.FilterOperator;

public class CustomFilters {

	public static FilterOperator IN = new FilterOperator("IN") {
		@Override
		public boolean matches(Object value1, Object value2) {
			throw new UnsupportedOperationException(); // handle differently
		}
	};

}

File: fm-domain/src/main/java/io/zeitwert/dddrive/persist/util/CustomFilters.kt
package io.zeitwert.dddrive.persist.util

import io.crnk.core.queryspec.FilterOperator

object CustomFilters {

	@JvmField
	var IN: FilterOperator = object : FilterOperator("IN") {
		override fun matches(
			value1: Any?,
			value2: Any?,
		): Boolean {
			throw UnsupportedOperationException() // handle differently
		}
	}

}
